### ISSUE TITLE
v2.0.7 breaks with Python <= 3.7

### ISSUE THREAD

**Issue by marc-etienne** [censys/censys-python#170]
**Describe the bug**
The following commit introduced a dependency on Python 3.8: b1cfe438.

`typing.Protocol` is available only since Python 3.8 (see https://docs.python.org/3/library/typing.html#typing.Protocol)

However, censys-python package is marked as compatible with Python 3.6+.

**To Reproduce**
1. Install censys-python
2. `import censys.common.base`

```
File "...\site-packages\censys\common\base.py", line 6, in <module>
    from typing import Any, Callable, List, Optional, Protocol, Type
ImportError: cannot import name 'Protocol' from 'typing' (...\typing.pyc)
```

**Expected behavior**
No exceptions

**Device (please complete the following information):**
 - OS [e.g. macOS]: Windows 10 (2004)
 - Censys Python Version [e.g. 1.1.0]: 2.0.7
 - Shell [e.g. bash, zsh]: cmd
 - Python: 3.7.9

**Comment by thehappydinoa** [comment]
Hi @marc-etienne,

Thank you very much for reporting this issue. I have begun to add additional tests for Python 3.6 and have created an MR to resolve this issue by switching it to a `Callable`.

**Comment by thehappydinoa** [comment]
Just following up, I just released `v2.0.8` to upgrade run `pip install --upgrade censys`.

**Comment by marc-etienne** [comment]
Thank you! That was fast 😄
